### PR TITLE
feat(KFLUXDP-104): Add a new task that get sealights references from a build pipeline

### PR DESCRIPTION
**Pull request by flacatus**
Create a new task that retreieves sealights metadata  from a build pipeline like:
* Build Session ID
* Container Image in case is golang
* Build Name
* The instrumented OCI artifact

### CODE REVIEW

File: docs/qe-available-tasks/sealights/sealights-get-refs.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Sealights Get Refs Tekton Task
```
Comment (ascerra) [inline docs/qe-available-tasks/sealights/sealights-get-refs.md:1]: Can we add dependencies to this task?

Like what tasks need to happening before we are able to run this?

Comment (flacatus) [reply docs/qe-available-tasks/sealights/sealights-get-refs.md:1]: Yes. Good point. I will add some examples

Comment (flacatus) [reply docs/qe-available-tasks/sealights/sealights-get-refs.md:1]: done

**Review by psturc — APPROVED** [review]

File: docs/qe-available-tasks/sealights/sealights-get-refs.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+|-------------------------------|---------------------------------------------------------------------------------|
+| `sealights-source-artifact`  | The source code OCI artifact that was instrumented with sealights.             |
+| `sealights-bsid`             | The Build Session ID (BSID) generated during the build for sealights.          |
+| `sealights-container-image`  | The container image used in the Sealights instrumentation process.             |
+| `sealights-build-name`       | The build name generated by sealights instrumentation.                         |
+| `container-image`            | The container image built from the specified Git revision without Sealights instrumentation. |
+
+## Usage Example
+
+Here's an example of how to use the `sealights-get-refs` Task in a Tekton Pipeline:
```
Comment (ascerra) [inline docs/qe-available-tasks/sealights/sealights-get-refs.md:33]: Is this meant to run only in the integration pipeline and is it meant to run after test-metadata?

Comment (flacatus) [reply docs/qe-available-tasks/sealights/sealights-get-refs.md:33]: I will say integration tests. Let me add it to the docs.

**Pull request merged**